Fix logger initialization in patron-cli

## Is your feature request related to a problem? Please describe

Not a feature request, rather than a regression fix.

## Is this a bug? Please provide steps to reproduce, a failing test etc.

```
$ cd ~/Beat-repos/patron/cmd/patron
$ go build
$ ./patron
$ ./patron -m "foobar" -p $PWD/foobar
$ cd foobar/
$ go run cmd/foobar/main.go
# command-line-arguments
cmd/foobar/main.go:19:9: undefined: patron.SetupLogger
```

## Describe the solution

The SetupLogger function was renamed to SetupLogging in #159, but the bootstrap CLI was not updated to reflect that change correctly.

```
$ cd ~/Beat-repos/patron/cmd/patron
$ go vuild
$ ./patron 
$ ./patron -m "foobar" -p $PWD/foobar
$ go run cmd/foobar/main.go
{"lvl":"info","host":"Paschalis-Tsilias.local","srv":"foobar","ver":"dev","time":"2020-09-15T13:02:03.083225+03:00","src":"patron/service.go:318","msg":"setting up default tracing 0.0.0.0:6831, probabilistic with param 0.0"}
{"lvl":"info","host":"Paschalis-Tsilias.local","srv":"foobar","ver":"dev","time":"2020-09-15T13:02:03.186471+03:00","src":"patron/service.go:323","msg":"creating default HTTP component at port 50000"}
...
```

## Additional context

<!-- OPTIONAL Add any other context or screenshots about the feature request here. -->

--- a/cmd/patron/main.go
+++ b/cmd/patron/main.go
@@ -246,7 +246,7 @@ var (
 func main() {
 	name := "{{ .Name}}"
 
-	err := patron.SetupLogger(name, version)
+	err := patron.SetupLogging(name, version)
 	if err != nil {
 		fmt.Printf("failed to set up logging: %v", err)
 		os.Exit(1)

--- a/cmd/patron/main_test.go
+++ b/cmd/patron/main_test.go
@@ -86,7 +86,7 @@ var (
 func main() {
 	name := "name"
 
-	err := patron.SetupLogger(name, version)
+	err := patron.SetupLogging(name, version)
 	if err != nil {
 		fmt.Printf("failed to set up logging: %v", err)
 		os.Exit(1)

